### DISCUSSION TITLE
Will there be more divisions?

### DISCUSSION THREAD

**Discussion by nicky-rodriguez** [EJL3/Plants_V_Zombies#1] [Q&A] [answered]
This project is super good, impressed. I just wanted to ask whether you will add more divisions as it has only one it seems(Adventure).

**Comment by EJL3** [comment, marked as answer]
Thank you for the compliment. As the question you asked, I am not gonna add anymore divisions (really sorry). The whole purpose of the project is to mimic some parts of the game to make a clear idea of how it works. As a matter of fact i will be adding more characters in the coming updates.

**Comment by vukhoinguyennguyen1902** [comment]
how can i wakeup the mushroom?
